pkg/kubectl/resource/selector.go

```diff
@@ -50,11 +50,10 @@ func (r *Selector) Visit(fn VisitorFunc) error {
 	if err != nil {
 		if errors.IsBadRequest(err) || errors.IsNotFound(err) {
```

Comment:
@pweil-, thinking about this more, I'd rather not mask the original error... can you leave the logging as-is, and just return the original error?

The following in `serviceErrorHandler` surfaces a not found error nicely for resources the server knows about...
```
errorJSON(apierrors.NewGenericServerResponse(serviceErr.Code, requestInfo.Verb, requestInfo.Kind, requestInfo.Name, "", 0, false), codec, response.ResponseWriter)
```

That should cover the "disabled feature" case where the types are known, and the API endpoint is simply inactive. We could look into improving how unknown types are surfaced there